### DISCUSSION TITLE
Progress bar

### DISCUSSION THREAD

**Discussion by sang-yeo** [twilio-labs/paste#540] [General] [answered]
Hello there, what is an alternative component(s) to using progress bar component? I used to see the progress bar component from old Paste site but I couldn't find it anymore.

**Comment by aayushpi** [comment, marked as answer]
Progress bars are on our roadmap, but not high priority based on prioritization across BUs. Recommend bringing your UI need to office hours alongside an engineer: it should be possible to build a great progress bar using our primitives that should solve for your needs till we have a component.
